CamMerrill/Shackelford
# Created destructible meshes for full plate and mouse bone

## Summary
Created the following
- Two new destructibles meshes `Meshes/DM_MouseBone` and `Meshes/DM_PlateA`
- Two new blueprints under `Blueprints/GrabbableObjects/Artifacts`
    - `BP_Plate_Destructible`
    - `BP_MouseBone_Destructible`

## Testing
1. Load a map and place a trowel `BP_Trowel_Example`
1. Place a `BP_Plate_Destructible` or `BP_MouseBone_Destructible` object in the scene
    - Note: You'll need to scale the bone down to 0.02 in x, y, and z.
1. Hit the artifacts with the trowel a few times and ensure they fracture and break (you can tell if physics starts being applied like gravity, etc.)

## Notes
- Currently works with `BP_Trowel_Example`
- Not sure how grabbable will work with the broken objects, but that's a later problem.
